agent/php_user_instrument.c

```diff
@@ -303,13 +362,6 @@ static nruserfn_t* nr_php_user_wraprec_create_named(const char* full_name,
   int klass_len;
   nruserfn_t* wraprec;
 
-  if (0 == full_name) {
```

Comment:
Since it's no longer validating `full_name` here, can you please leave a comment that this function expects `full_name` to be a prevalidated non-null, non-zero length string that doesn't end in `:`?

Reply:
See 575aa95bd52a225774fb2428fa7520d64fdea37f.